core/java/iesi-core/src/main/java/io/metadew/iesi/common/crypto/FrameworkCrypto.java

```diff
@@ -19,15 +22,28 @@
 
     private static FrameworkCrypto INSTANCE;
 
-    public static FrameworkCrypto getInstance() {
+    public static FrameworkCrypto getInstance() throws Exception {
         if (INSTANCE == null) {
             INSTANCE = new FrameworkCrypto();
         }
         return INSTANCE;
     }
 
-    private FrameworkCrypto() {
-        this.aes = new AESEncryptBasic("c7c1e47391154a6a");
+    private FrameworkCrypto() throws Exception {
```

Comment:
Where and what Exception is thrown exactly?
If it is related to the instantiation of the Java Keystore, you can transform it to a RuntimeException as it will be a critical error. This will allow you to remove all the 'Exception' added to the other methods.